locator/token_metadata.cc

```diff
@@ -570,8 +570,7 @@ std::unordered_map<inet_address, host_id> token_metadata_impl::get_endpoint_to_h
     std::unordered_map<inet_address, host_id> map;
```

Comment:
I recall we discussed renaming `get_endpoint_to_host_id_map_for_reading`, did you abandon the idea?

Reply:
Lets do that separately, since this is decoupled.

Reply:
I made this name change!